draft-ietf-quic-recovery.md

```diff
@@ -362,10 +362,10 @@ Using max(SRTT, latest_RTT) protects from the two following cases:
 * the latest RTT sample is higher than the SRTT, perhaps due to a sustained
   increase in the actual RTT, but the smoothed SRTT has not yet caught up.
 
-Implementers MAY experiment with using other reordering thresholds, including
-absolute thresholds, bearing in mind that a lower multiplier reduces reordering
-resilience and increases spurious retransmissions, and a higher multiplier
-increases loss detection delay.
+Implementers MAY experiment with including RTT variance in the threshold, or
```

Comment:
I don't know how to check for reordering with PING. If I send PINGs in burst from Denmark to China I see maybe 30% packet loss, the same if I send packets in short intervals. I think the loss might be because the ping tool does not wait for a response after at sends the next PING and opts for a timeout loss instead. This from Denmark to Beijing with variance of 200-500ms.

Reply:
To my knowledge, you can't.  Which is why PING doesn't tell you anything about network reordering, unfortunately.  It does tell you a lot about RTT and RTTVar, though.

Reply:
To make this slightly harder still, Chromium's early experiments showed that loss was not correlated, but when an application is using the network, evidence shows loss is very correlated.  So I don't think we can understand the prevalence and nature of reordering in a network without using the network in a realistic way, such as HTTP.

Reply:
I didn't get past the double slit experiment.

Reply:
@ianswett: I don't think the long list of RTT measurements on this issue are making the necessary point, and I don't have any reason to believe that high RTT variability has anything to do with reordering. That said, it's definitely possible for a network to have high variability and reordering. They don't need to be correlated, they just need to happen together.

Reply:
Theoretically it isn't hard to imagine a router taking a different path once it realises a preferred path is congested. That can easily have one packet sitting in queue while another packet races past on an otherwise longer path. The preferred might suddenly unclog leading a third packet to overtake the second packet on the longer path.

Whether this happens in real life is not explained by hypothesis.